blueprint/core/blueprint_View.cpp

```diff
@@ -53,6 +53,11 @@ namespace blueprint
             _refId = juce::Identifier(value.toString());
     }
 
+    const juce::var& View::getProperty(const juce::Identifier& name) const
```

Comment:
I think we can omit this method now, right?

Reply:
Yeah fair point. Will remove.